Source/UI/Controllers/AddressViewController.swift

```diff
@@ -181,8 +181,8 @@ public class AddressViewController: UIViewController,
 
     // MARK: - CountrySelectionViewControllerDelegate
 
-    /// Executed when an user select a country.
-    public func onCountrySelected(country: Country) {
+    /// Executed when a user select a country.
+    public func onCountrySelected(country: Country, tag: Int) {
```

Comment:
does this needs to be public ?

Reply:
This is old billing form UI. I think, it will be removed after approving new billing form UI